layouts/_default/list.html

```diff
@@ -6,7 +6,7 @@ <h3 style="margin-bottom:0">Filtering for "{{ .Title }}"</h3>
     <a href="{{ "blog" | relURL }}">Remove filter</a>
   </small>
   {{ end }}
-  <ul class="blog-posts">
+  <ul class="posts">
```

Comment:
```suggestion
  <ul class="blog-posts">
```

I'd like to keep the `blog`-prefix to make syncing the `CSS` with https://bearblog.dev/ a simple copy & paste.